PropertyChanged.Fody.Analyzer/CodeBuilder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Text;
+
+internal class CodeBuilder
+{
+    private readonly StringBuilder _stringBuilder = new();
```

Comment:
You could leverage `IndentedTextWriter` here.

Reply:
looks like some overhead to replace one line of code with five.

Reply:
I meant this could have handled indents for you, but as you wish.